src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -620,6 +646,399 @@ components:
         - STEP_STARTED
         - STEP_COMPLETED
 
+    CreateCiagInductionRequest:
+      title: CreateCiagInductionRequest
+      type: object
+      description: A request to create a CIAG Induction. Migrated from hmpps-ciag-careers-induction-api.
+      properties:
+        requestDTO:
+          $ref: '#/components/schemas/CreateCiagInductionRequestData'
+        oauth2User:
+          type: string
+
+    CreateCiagInductionRequestData:
+      title: CreateCiagInductionRequestData
+      description: The request data to create a CIAG Induction. Migrated from hmpps-ciag-careers-induction-api.
+      type: object
+      properties:
+        prisonId:
+          type: string
+          description: The ID of the Prison that that Prisoner is currently at.
+        prisonName:
+          type: string
+          description: The name of the Prison that that Prisoner is currently at.
+        hopingToGetWork:
+          $ref: '#/components/schemas/HopingToWork'
+        reasonToNotGetWorkOther:
+          type: string
+          description: The reason that is given when the Prisoner does not want to work. This is mandatory when 'reasonToNotGetWork' is set to 'OTHER'.
```

Comment:
This might be a good place to use one of those custom constraints; (like where we used `x-constraints: 'NotInPast'` elsewhere in the spec and then wrote `NotInPastValidator`)
Defo not suggesting we do it in this PR, just saying it might be a good fit when we get to the implementation and the validations etc 👍

Reply:
Also, as and when we get into the implementation and need to think about validation, we should probably refer to this class `CIAGValidationService`
I don't think they are doing validation anywhere else, and certainly not with standard JSR310 type annotations, which I think we should try to do if we can 👍